### ISSUE TITLE
Inline Chat Does Not Work In Accessibility Mode: Progress Reporting is Only Supported For Live Sessions

### ISSUE THREAD

**Issue by hediet** [microsoft/vscode#188750]
Testing #188537

![Code_-_Insiders_wVCfJlfUdT](https://github.com/microsoft/vscode/assets/2931520/f4c730c1-2c44-401b-947d-cd1f1514e4ce)

**Comment by roblourens** [comment]
Not specific to accessibility mode

**Comment by roblourens** [comment]
Nevermind, actually it does work fine with screenreader mode disabled

**Comment by roblourens** [comment]
Ah I see, you can also repro it by setting the chat mode setting to Preview- that's what "live" refers to. I assume that somewhere we default to that mode when screenreader mode is enabled. And previously we probably didn't call `progress.report` all the time. Hopefully one of the people on the issue know where this needs to be addressed.

**Comment by alexdima** [comment]
The fix will be available in around 45 minutes with `0.5.2023072601`
